bundles/org.openhab.binding.homeconnect/README.md

```diff
@@ -135,7 +135,7 @@ The Home Connect bridge can be configured in the openHAB UI as follows:
 
 ## Examples: File based configuration
 
-If you prefer to configure everything via file instead of openHAB UI, here are some examples.
+If you prefer to configure everything via file instead of openHAB UI, here are some examples. In order to get your `haId` you might use the discover feature in paper UI, copy the appliance's ID and paste it into your `*.things`.
```

Comment:
Since openHAB 3.0 there is no paper UI (we now have Main UI):
```suggestion
If you prefer to configure everything via file instead of openHAB UI, here are some examples. In order to get your `haId` you might use the discover feature in UI, copy the appliance's ID and paste it into your `*.things`.
```